routes/api.js

```diff
@@ -219,4 +219,20 @@ router.post("/setcommand/:detector", checkKey, function(req, res) {
   }).catch((err) => res.json({message: err.message}));
 });
 
+function GetSystemMonitorStatus(collection, host, callback) {
```

Comment:
`callback` doesn't look like it gets used. If we were better javascript coders we would actually differentiate between returning a Promise and using the callback function, but that takes effort.

Reply:
I wasn't sure what it was for so I left it there. Now it's gone